### DISCUSSION TITLE
Prometheus settings on Alpha 9 v2

### DISCUSSION THREAD

**Discussion by mavacpjm** [nzymedefense/nzyme#1002] [General]
Question about the Prometheus REST API, I enabled with user and pass. Not seeing the metrics URL, is it a special URL ?

Thanks!

**Comment by lennartkoopmann** [comment]
Hi! Yes, it's a specific path/resource.

I will add instructions to the documentation tomorrow and let you know once it's online!

**Reply by mavacpjm** [reply]
Roger that, thanks!!!!

**Reply by lennartkoopmann** [reply]
Here we go! https://docs.nzyme.org/configuration/monitoring/metrics-exporters/

Let me know if anything is missing.

**Reply by mavacpjm** [reply]
You Rock, thanks trying it now! Will do!

**Reply by mavacpjm** [reply]
Works Perfect, Awesome - thanks!

**Reply by mavacpjm** [reply]
One quick question, I used heavily the GRAYLOG integration to visualize nzyme in a SIEM. Do you plan to add that back in, or are you focused on Prometheus for this function. I am not seeing schema for prometheus that gives the insights that old nzyme did with graylog. Let me know, thanks again!

**Reply by lennartkoopmann** [reply]
Awesome! You are welcome!

SIEM/Log Management integrations will be part of the upcoming "Forwarders" feature. So far I have a working OpenSearch integration prototype.

**Reply by mavacpjm** [reply]
Awesome - Fantastic, thanks !
